content/posts/customize-versions-urls.md

```diff
@@ -0,0 +1,20 @@
+title: Customize the URLs for your versions
+date: May 23, 2025
+description: Announcing the ability to customize the URLs your versions are served from.
+category: Feature announcement
+tags: hosting
+authors: Santos Gallegos
+status: published
+image: /images/update-version-slug.png
+
+Versions are a core part of Read the Docs.
+They let you host multiple versions of your documentation.
+Each version has its own unique URL identifier (slug),
```

Comment:
I like that this isn't just referring to this as a "slug" :+1:

This terminology is much clearer than "slug", even if it is still technical terms.